Cargo.toml

```diff
@@ -16,7 +16,7 @@ name = "sdl2"
 path = "src/sdl2/lib.rs"
 
 [dependencies]
-bitflags = "1.2.1"
+bitflags = "2.5"
```

Comment:
This is a breaking change because this crate has publicly accessible `bitflags!` generated structs, can you document this in the changelog?

Please also refer users to bitflag's changelog entry of the 2.0.0 breaking changes: https://github.com/bitflags/bitflags/blob/main/CHANGELOG.md#200